doc/md/Installation_Instructions/Mac-OS-X-Compile-From-Source-Instructions.md

```diff
@@ -0,0 +1,70 @@
+<a id="Top"></a>
+
+# Mac OS X - Compilation from source instructions
+
+# Table of Contents
+- [Mac OS X - Compilation from source instructions](x#mac-os-x---compilation-from-source-instructions)
+- [Table of Contents](#table-of-contents)
+  - [Installing build prerequisites via Homebrew](#installing-build-prerequisites-via-homebrew)
+  - [Configure the build](#configure-the-build)
+  - [Compilation from source](#compilation-from-source)
+
+## Installing build prerequisites via Homebrew
+^[Top](#top)
+
+We need to install the ``openssl`` library with ``brew``:
+```bash
+brew install openssl
+```
+To run the local install script below, we also require the GNU versions 
+of several core Unix utilities:
+```bash
+brew install coreutils
+```
+
+## Configure the build
+^[Top](#top)
+
+Clone the repository by running the following:
+```bash
+git clone https://github.com/RfidResearchGroup/proxmark3.git
+cd proxmark3
+```
+Configure the build by editing ``Makefile.platform``:
+```bash
+cp Makefile.platform.sample Makefile.platform
+vim Makefile.platform
+```
+
+## Compilation from source
+^[Top](#top)
+
+The following command has been shown to work on MacOS Big Sur with the 
+default Compiler Tools that comes installed on this release of the platform:
+```bash
+make clean && CFLAGS="-I /usr/local/opt/openssl/include" make
```

Comment:
I believe @doegox  pushed a fix for building on OSX;   so the CLFAGS part here shouldn't be needed.